lib/jsdom/living/helpers/svg/render.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+const { SVG_NS } = require("../namespaces");
+
+// https://svgwg.org/svg2-draft/render.html#TermNeverRenderedElement
+const neverRenderedElements = new Set([
+  "clipPath",
+  "defs",
+  "desc",
+  "linearGradient",
+  "marker",
+  "mask",
+  "metadata",
+  "pattern",
+  "radialGradient",
+  "script",
+  "style",
+  "title",
+  "symbol"
+]);
+
+// https://svgwg.org/svg2-draft/render.html#Rendered-vs-NonRendered
+exports.isRenderedElement = function isRenderedElement(elImpl) {
+  if (neverRenderedElements.has(elImpl._localName)) {
+    return false;
+  }
+
+  if (elImpl.getAttributeNS(null, "display") === "none") {
```

Comment:
display is not an attribute; it is a CSS property. I'll remove this since we don't have the counterpart check for HTML.

Reply:
display is still an attribute in SVG, and you are right the spec says **computed style** value of none for the display **property**.